config/sets/doctrine-orm-213.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\MethodCall\RenameMethodRector;
+use Rector\Renaming\ValueObject\MethodCallRename;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(
+        RenameMethodRector::class,
+        [
+            // @see https://github.com/doctrine/orm/issues/9875
+            new MethodCallRename(
+                'Doctrine\ORM\Event\PreFlushEventArgs',
```

Comment:
- OnClearEventArgs
- LifecycleEventArgs
- OnFlushEventArgs
- PostFlushEventArgs
- PreFlushEventArgs

Seems also have this deprecated method and should also be added?

Reply:
I added these events as well as test fixtures